### PR TITLE
Move Java11 samples

### PR DESCRIPTION
**Pull request by averikitsch**
Moved and tested from Ludo's directory (https://github.com/ludoch/samples/tree/master/java11)

### CODE REVIEW

**Comment by averikitsch** [comment]
@dzlier-gcp @kurtisvg @lesv would anyone be willing to review these samples moved from Java 8?

File: appengine-java11/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+[ae-docs]: https://cloud.google.com/appengine/docs/java/
+
+## Prerequisites
+
+**Private Alpha**
+To access the Java 11 Google App Engine standard runtime, you'll need to apply
+to become part of the App Engine [Alpha program](https://docs.google.com/forms/d/e/1FAIpQLSf5uE5eknJjFEmcVBI6sMitBU0QQ1LX_J7VrA_OTQabo6EEEw/viewform).
+
+
+### Download Maven
```
Comment (lesv) [inline appengine-java11/README.md]: You might consider also doing Gradle for all of these.

File: appengine-java11/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+These samples use the [Apache Maven][maven] build system. Before getting
+started, be sure to [download][maven-download] and [install][maven-install] it.
+When you use Maven as described here, it will automatically download the needed
+client libraries.
+
+[maven]: https://maven.apache.org
+[maven-download]: https://maven.apache.org/download.cgi
+[maven-install]: https://maven.apache.org/install.html
+
+### Create a Project in the Google Cloud Platform Console
```
Comment (lesv) [inline appengine-java11/README.md]: Is there an API that needs to be enabled?

File: appengine-java11/appengine-simple-jetty-main/src/main/java/com/example/appengine/demo/jettymain/Main.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    // The WebAppContext is the interface to provide configuration for a web
+    // application. In this example, the context path is being set to "/" so
+    // it is suitable for serving root context requests.
+    WebAppContext webapp = new WebAppContext();
+    webapp.setContextPath("/");
+    webapp.setWar(args[0]);
+    ClassList classlist = ClassList.setServerDefault(server);
+
+    // Enable Annotation Scanning.
+    classlist.addBefore(
```
Comment (lesv) [inline appengine-java11/appengine-simple-jetty-main/src/main/java/com/example/appengine/demo/jettymain/Main.java]: Useful, but should you also run the jetty quickstart in your pom?  https://www.eclipse.org/jetty/documentation/9.4.x/quickstart-webapp.html

File: appengine-java11/README.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+   export JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64
+```
+
+## Set up
+Build the samples with maven:
+```
+mvn clean package
+```
+
+### appengine-simple-jetty-main
```
Comment (lesv) [inline appengine-java11/README.md]: All of these samples could also be used w/ Cloud Run w/ minor modifications.  You might consider releasing them both ways.

File: appengine-java11/bigquery/pom.xml [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+
+  <!--
+    The parent pom defines common style checks and testing strategies for our samples.
+    Removing or replacing it should not effect the execution of the samples in anyway.
+  -->
+  <!-- <parent>
+    <groupId>com.google.cloud.samples</groupId>
+    <artifactId>shared-configuration</artifactId>
+    <version>1.0.10</version>
+  </parent> -->
```
Comment (lesv) [inline appengine-java11/bigquery/pom.xml]: Then why have you commented it out?

File: appengine-java11/bigtable/src/main/java/com/example/bigtable/Main.java [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.example.bigtable;
+
+import com.google.cloud.bigtable.hbase.BigtableConfiguration;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import org.apache.hadoop.hbase.HColumnDescriptor;
+import org.apache.hadoop.hbase.HTableDescriptor;
```
Comment (lesv) [inline appengine-java11/bigtable/src/main/java/com/example/bigtable/Main.java]: Isn't there a newer Bigtable Java API Client Library that doesn't use HBase?  Won't that be easier to use?

Comment (lesv) [reply appengine-java11/bigtable/src/main/java/com/example/bigtable/Main.java]: @sduskis FYI

File: appengine-java11/gaeinfo/pom.xml [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+              <outputDirectory>${project.build.directory}/appengine-staging/cprof</outputDirectory>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <groupId>com.google.cloud.tools</groupId>
+        <artifactId>appengine-maven-plugin</artifactId>
+        <version>2.0.0-rc5</version>
```
Comment (lesv) [inline appengine-java11/gaeinfo/pom.xml]: rc5?  Really?

File: appengine-java11/README.md (hunk 1 above)
Comment (averikitsch) [reply appengine-java11/README.md]: On my to do list.

File: appengine-java11/gaeinfo/pom.xml (hunk 7 above)
Comment (averikitsch) [reply appengine-java11/gaeinfo/pom.xml]: That's the latest version: https://mvnrepository.com/artifact/com.google.cloud.tools/appengine-maven-plugin

File: appengine-java11/bigquery/pom.xml (hunk 5 above)
Comment (averikitsch) [reply appengine-java11/bigquery/pom.xml]: Currently doesn't compile with the parent. I'm looking into the issue - looks like its a compatibility issue with error prone.

File: appengine-java11/appengine-simple-jetty-main/src/main/java/com/example/appengine/demo/jettymain/Main.java (hunk 3 above)
Comment (averikitsch) [reply appengine-java11/appengine-simple-jetty-main/src/main/java/com/example/appengine/demo/jettymain/Main.java]: I see the benefits of using the jetty quickstart, but I'm not sure how to use it.

File: appengine-java11/gaeinfo/pom.xml (hunk 7 above)
Comment (lesv) [reply appengine-java11/gaeinfo/pom.xml]: You might wish to ask them when the plan to release it if you are including it in samples.

**Comment by averikitsch** [comment]
I have investigated having a cleaner story for the jetty server, but this seems the simplest for now. It has shown me that mostly a Spring Boot framework would be the easiest way to get started with Java 11. Initially, these samples were being moved over so we could have a handful of examples to look at. With all of that info, is it worth having these sample and should we rethink what samples we are providing?

**Pull request merged**